java/client/test/org/openqa/selenium/chrome/ChromeOptionsFunctionalTest.java

```diff
@@ -81,6 +83,7 @@ public void canSetAcceptInsecureCerts() {
   }
 
   @NeedsLocalEnvironment
+  @NotYetImplemented(reason="Needs Resource: third_party/chrome_ext/backspace.crx/")
```

Comment:
If you rebase on `HEAD` of `trunk` this should now be fixed and working as expected.